src/MyHealth/symptoms.ts

```diff
@@ -44,53 +42,3 @@ export const determineHealthAssessment = (
   }
   return HealthAssessment.NotAtRisk
 }
-
-type WithADate = { date: Posix }
-const compareDatesAscending = (
```

Comment:
Did we end up sorting the logs in any form? Should we have them sorted? It seems on the screenshot that they are not.